portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,18 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+// Fetch a greeting from server and add it to page.
+async function getGreeting() {
+
+  //const response = await fetch('/data');
+  //const greeting = await response.text();
+  //document.getElementById('greeting-container').innerHTML = greeting;
+
+  fetch('/data').then(response => response.json()).then((greetings) => {
+    const greetingElem = document.getElementById('greeting-container');
+    var index = Math.floor(Math.random() * 10) % 3;
```

Comment:
a few things about this line:

1. use `const index`
2. although what you have works, getting a random integer would typically be done by `Math.floor(Math.random() *  3)`
3. the code depends on the list only having 3 elements. This is an error-prone construction, in that if you were to add (or delete) a member from the list and forgot to update this code too, it would not behave as expected. Is there something better you can do here?